index.html

```diff
@@ -389,6 +415,7 @@ <h4>Powered By <a href="https://github.com/omaciel/capacitator" target="_blank"
                     updateTeamSize();
                     calculateSprintDays()
                     calculateTotalTeamDays();
+                    updateTeamMembersStoryPointCapacity();
```

Comment:
This is in the `loadTemplateFromFile()` function (although this `diff` suggests it's in `saveTemplateAsJSON()`.

I don't store the `capacity` field in the export file and it is recalculated when loaded.